.github/workflows/build.yml

```diff
@@ -0,0 +1,43 @@
+name: PolkadexTEE-worker CI
+
+on:
+  push:
+    branches: [ 'master','develop' ]
+  pull_request:
+    branches: [ 'master','develop' ]
+
+jobs:
+  build-polkadexTEE:
+    runs-on: ubuntu-latest
+    container: "scssubstratee/substratee_dev:1804-2.12-1.1.3-001"
+    steps:
+      - uses: actions/checkout@v2
+#      - name: Cache Rust Dependecies
+#        uses: actions/cache@v2
+#        with:
+#          path: |
+#            ~/.cargo/registry
+#            ~/.cargo/git
+#            target
+#            enclave/target
+#            bin
+#            bin2
+#          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+      - name: Install Rust and WASM Environment
+        run: ./ci/install_rust.sh
+      - name: Information
+        run: cargo --version && rustup show && env
+      - name: Build Project
+        run: SGX_MODE=SW make
+      - name: Test Client
+        run: cd client && cargo test
+      - name: Test Worker
+        run: cd worker && cargo test
+      - name: Test Enclave
+        run: cd bin && ./substratee-worker test --unit
```

Comment:
does this actually return a non-zero value if tests fail? @haerdib 
In order for CI to fail we'd need to return an error value....

Reply:
Currently the code panics when a tests fails due to the unwrap() [here](https://github.com/Polkadex-Substrate/polkadexTEE-worker/blob/master/worker/src/tests/mod.rs#L36):

```
test result FAILED. 46 tested, 45 passed, 1 failed
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: SGX_ERROR_UNEXPECTED', worker/src/tests/mod.rs:36:27
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace 
```
Or how would you approach this?

Reply:
panic should do the trick